react/src/lib/EventOverlay/index.js

```diff
@@ -686,15 +686,14 @@ class EventOverlay extends React.Component {
         break;
       case 'left':
         if (!scrollParentDims.left && !transformParentDims) {
-          if (
-            arrowWidth + offsetWidth + elementDims.width + anchorPosition.left >
-            anchorPosition.left
-          ) {
+          targetNode.style.right = `${documentRight -
+            anchorPosition.left +
+            arrowWidth +
+            offsetWidth}px`;
+          if ( elementDims.left - arrowWidth < 0 ) {
             targetNode.style.left = `${arrowWidth}px`;
-            targetNode.style.right = `${documentRight -
-              anchorPosition.left +
-              arrowWidth +
-              offsetWidth}px`;
+          } else {
+            targetNode.style.left = 'inherit';
```

Comment:
Previously the left edge of the overlay would always be at the left edge of the screen. This else takes care of the case where there is enough room to place the tooltip properly, i.e. it is not too far left